File: graphql-dgs-codegen-core/src/integTest/java/com/netflix/graphql/dgs/codegen/java/testcases/projections/projectionWithTypeAndArgs/expected/DgsConstants.java
package com.netflix.graphql.dgs.codegen.java.testcases.projections.projectionWithTypeAndArgs.expected;

import java.lang.String;

public class DgsConstants {
  public static final String QUERY_TYPE = "Query";

  public static class QUERY {
    public static final String TYPE_NAME = "Query";

    public static final String Person = "person";

    public static class PERSON_INPUT_ARGUMENT {
      public static final String A1 = "a1";

      public static final String A2 = "a2";

      public static final String A3 = "a3";
    }
  }

  public static class EMPLOYEE {
    public static final String TYPE_NAME = "Employee";

    public static final String Firstname = "firstname";

    public static final String Company = "company";
  }

  public static class I {
    public static final String TYPE_NAME = "I";

    public static final String Arg = "arg";
  }

  public static class PERSON {
    public static final String TYPE_NAME = "Person";

    public static final String Firstname = "firstname";
  }
}

File: graphql-dgs-codegen-core/src/integTest/kotlin/com/netflix/graphql/dgs/codegen/cases/projectionWithTypeAndArgs/expected/DgsConstants.kt
package com.netflix.graphql.dgs.codegen.cases.projectionWithTypeAndArgs.expected

import kotlin.String

@Generated
public object DgsConstants {
  public const val QUERY_TYPE: String = "Query"

  @Generated
  public object QUERY {
    public const val TYPE_NAME: String = "Query"

    public const val Person: String = "person"

    @Generated
    public object PERSON_INPUT_ARGUMENT {
      public const val A1: String = "a1"

      public const val A2: String = "a2"

      public const val A3: String = "a3"
    }
  }

  @Generated
  public object EMPLOYEE {
    public const val TYPE_NAME: String = "Employee"

    public const val Firstname: String = "firstname"

    public const val Company: String = "company"
  }

  @Generated
  public object I {
    public const val TYPE_NAME: String = "I"

    public const val Arg: String = "arg"
  }

  @Generated
  public object PERSON {
    public const val TYPE_NAME: String = "Person"

    public const val Firstname: String = "firstname"
  }
}
